transports/http/endpoints/api/tips/tips_endpoints_test.go

```diff
@@ -127,52 +126,52 @@ func TestGetTipLongest(t *testing.T) {
 	})
 }
 
-func TestPruneTip(t *testing.T) {
-	t.Run("failure when authorization on and empty auth header", func(t *testing.T) {
-		// given
-		pulse, cleanup := testpulse.NewTestPulse(t)
-		defer cleanup()
-		expected_result := struct {
-			code int
-			body []byte
-		}{
-			code: http.StatusUnauthorized,
-			body: []byte("\"empty auth header\""),
-		}
-
-		// when
-		res := pulse.Api().Call(pruneTip("123"))
-
-		// then
-		assert.Equal(t, res.Code, expected_result.code)
-		body, _ := io.ReadAll(res.Body)
-		assert.EqualBytes(t, body, expected_result.body)
-	})
-
-	t.Run("success", func(t *testing.T) {
-		// given
-		pulse, cleanup := testpulse.NewTestPulse(t, testpulse.WithLongestChain(), testpulse.WithApiAuthorizationDisabled())
-		defer cleanup()
-		expected_result := struct {
-			code int
-			body string
-		}{
-			code: http.StatusOK,
-			body: "",
-		}
-
-		// when
-		res := pulse.Api().Call(pruneTip("123"))
-
-		// then
-		assert.Equal(t, res.Code, expected_result.code)
-
-		var tip string
-		json.NewDecoder(res.Body).Decode(&tip)
-
-		assert.Equal(t, tip, expected_result.body)
-	})
-}
+// func TestPruneTip(t *testing.T) {
```

Comment:
I would remove test as it will be not useful after refactor

Reply:
removed